documentation/CHANGELOG_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+# Changelog API Sub Project name
+
+<!--NOTE: in above title replace "API Sub Project name" with actual API Sub Project name and remove this line-->
+
+## Table of content
+
+- **[r1.1 - alpha release](#r11---alpha-release)**
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+The below sections record the changes for each API version in each release as follows:
+
+* for an alpha release, the delta with respect to the previous release
+* for the first release candidate, all changes since the last public release
+* for subsequent release candidate(s), only the delta to the previous release candidate
+* for a public release, the consolidated changes since the previous public release
+
+<!-- Repeat the below release section (header 1 and subsections) at the top of this file for each new release-->
+
+# r1.1 - alpha release
```

Comment:
I'm not sure if we should mention alpha and release candidate within the release title as this are already postfixes of the API versions. I propose that we have only "pre-release" and release numbers without postfix (the public ones).

Reply:
updated to just using pre-release and release in the titles

Reply:
added minimal info in the title. per above proposal. will update this ones resolved. The above title is now "r1.1 - alpha"